package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "issue-finder",
-  "version": "0.3.5",
+  "name": "issue-crawler",
+  "version": "0.0.1",
```

Comment:
theres a newer lockfile already, please update yours or remove it from the commit 👍